pulp_docker/app/models.py

```diff
@@ -123,6 +123,9 @@ class BlobManifestBlob(models.Model):
     manifest_blob = models.ForeignKey(
         ManifestBlob, related_name='manifest_blobs', on_delete=models.CASCADE)
 
+    class Meta:
+        unique_together = ('manifest', 'manifest_blob')
```

Comment:
After adding the viewsets, I found that the many-to-many relationships were being written multiple times. This made the serialization really ugly, and I wasn't sure it was working correctly, so I went ahead and fixed this problem since it was simple.

If preferred, I can break these changes into a different PR, but making the change makes it easier to check that the viewsets/serializers were working as expected